### ISSUE TITLE
[1.16] Is documentation outdated? 

### ISSUE THREAD

**Issue by iarspider** [IntelligenceModding/Advanced-Peripherals-Documentation#28]
### Descripe

First of all, thanks for making Turtles Useful Again! :) 

I'd like to point out that some entries in the 1.16 documentation are out of date. For example, the following methods for Weak Automata Core are missing: `getSuckCooldown`, `getDigCooldown`, `getUseOnBlockCooldown`. Instead, there is a single `getOperationCooldown(string name)` method.

Then, what exactly does `feedSoul` do? I guess it's for making other souls (husbandry/end)?

Also, maybe reword the `chargeTurtle` description by mentioning that this method charges turtle using FE container items? E.g., change the description to "Tries to charge turtle **using a FE container item (capacitor)** and returns amount of recEived fuel or errors".

[Overpowered automata](https://docs.srendi.de/1.16/metaphysics/overpowered_automata/) documentation is very vague (and probably misleading):
> If you try to perform any operation with this soul without fuel - upgrade will broke immediately.

If I'm reading [the code](https://github.com/Seniorendi/AdvancedPeripherals/blob/1.16/src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/peripheral/metaphysics/OverpoweredWeakAutomataCorePeripheral.java#L20) correctly, there is a flat chance that an upgrade will break, regardless of fuel level. Also, "upgrade will BREAK", not "upgrade will BROKE", but that's not that important.

### Steps to reproduce

1. Open the documentation
2. Try to follow it
3. Lie down
4. Try not to cry when your precious Overpowered Weak Automata Soul breaks
5. Cry

### Direct URL

https://docs.srendi.de/1.16/metaphysics/weak_automata/

**Comment by SirEndii** [comment]
Sorry, I didn't touch the documentation in a while

Thank you for the issue report, I'll see what I can do

**Comment by SirEdvin** [comment]
>
    Open the documentation
    Try to follow it
    Lie down
    Try not to cry when your precious Overpowered Weak Automata Soul breaks
    Cry

I hate myself, because this is my issue. I have changed this logic for simplification and ... well .... not updated documentation

Will be fixed

**Comment by SirEndii** [comment]
Should be solved with the latest documentation revamp
